simhash_db/elasticsearch_client.py

```diff
@@ -0,0 +1,194 @@
+#! /usr/bin/env python
+
+'''Code to connect to the ElasticSearch backend'''
+
+import struct
+import time
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionError
+from . import BaseClient
+import calendar
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import dateutil.parser
```

Comment:
You are not using the following imports
from elasticsearch.exceptions import ConnectionError
import calendar
from datetime import datetime
from dateutil.relativedelta import relativedelta
import dateutil.parser